### ISSUE TITLE
Mistakes or not?

### ISSUE THREAD

**Issue by highfield** [vvvmax/unitegallery#50]
Visual Studio warns for some potential issues in the "unitegallery.js" (not minimized) module. Please, check if the following are mistakes:
Function "scaleImageFitParent". There is a couple of assignments (?) which look like conditions instead:
`updateCss == true`

Function "onTouchEnd". Same as above:
`g_temp.isPanActive == false;`

Function "hideArrows". The if's trailing semicolon is probably wrong:
`if(isMouseInsideImage() == true);`

I temporarily patched my cached copy of the module.

**Comment by vvvmax** [comment]
HI, thank you (and visual studio :)
Fixed those mistakes, made commit at github. you can download new version
if you want.

**Comment by highfield** [comment]
Many thanks, but did you checked the last point as well? The latest commit keeps the suspect semicolon: https://github.com/vvvmax/unitegallery/blob/master/package/unitegallery/js/unitegallery.js#L21913
Just to validate it: JavaScript is tricky (too much for me, at least!).

**Comment by vvvmax** [comment]
yes, I fixed the last issue as well. weird. But, if the gallery is working
then it's ok, you don't have to worry about it.
